extensions/wikia/AdEngine/js/provider/btfBlocker.js

```diff
@@ -95,8 +96,12 @@ define('ext.wikia.adEngine.provider.btfBlocker', [
 			}
 		}
 
-		function shouldDelaySlotFillIn() {
-			return adContext.getContext().opts.delayBtf && !adBlockDetection.isBlocking()
+		function shouldDelaySlotFillIn(slotName) {
+			var isBlocking = adBlockDetection.isBlocking() || instartLogic.isBlocking(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the declaration of the unused 'isBlocking' variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedVariable)
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "isBlocking" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1854)